src/text.scss

```diff
@@ -0,0 +1,18 @@
+@import "./new-settings";
+@import './mixins';
+
+$block: #{$fd-namespace}-text;
+
+.#{$block} {
+  @include fd-reset();
+
+  font-weight: normal;
+  font-family: var(--sapFontFamily);
+  font-size: var(--sapFontSize);
+  color: var(--sapTextColor);
```

Comment:
```
  font-weight: normal;
  font-family: var(--sapFontFamily);
  font-size: var(--sapFontSize);
  color: var(--sapTextColor);
```
are already included in `fd-reset()`

Reply:
@jbadan Yeap, I saw it, but my idea was that these styles are specific for text component, from design guidelines.
And styles from `fd-reset()` are styles from `fd-reset()` they have different scopes, I mean if something will change in `fd-reset` It will affect the text component, but it shouldn't.

Reply:
We shouldn't repeat css - it needs to be removed.

Reply:
On another side, we should make components independent.

And a few other points:
- It's a common practice to overwrite styles from reset styles, we just got lucky that these style values are the same.
- It's a common practice inside an already created components.

Reply:
It is repeating styles, you're overwriting a value with the same value. It is only common to overwrite reset styles when the styles are different from the ones in the reset. The reset serves a purpose to not allow outside global styles to leak into our components.

Reply:
As I mentioned earlier, it's just a coincidence that this component and `fd-reset` styles have the same values.

We have an independent component with its styles, and we have a set of reset styles, it is two different scopes.
Reset styles - is a global set of rules and global scope.
And the text component has its own set of rules, with its own scop.

The last one has the main priority, and if something will be changed in `fd-reset`, by mistake or on purpose this shouldn't affect the main text component styles.

In my opinion, this is a more bulletproof approach, without unclear relations.

Reply:
Hi @EvillAlexei, the current approach of Fundamental-styles is to use the fd-reset and not to repeat the css. This set of rules is not specific only for the Text component, it's used in many other components dealing with text. Could you add the reset here and we can go on with the review?

Reply:
Hi @InnaAtanasova , I wasn't against `fd-reset` and it's already presented in the component.
I'm against removing scoped styles, named as repeated. I checked and found such behaviour in many existed components:
alerts.scss, menu.scss, pagination.scss and else. So as I understand it is or maybe it was ok to use styles in this way.

And I see a lot of sense in this approach, `fd-reset` - it's a set of reset styles, it's a common mixin, and it's could be changed easily, but this shouldn't affect components main styles, from design specification. But in this case, If I remove the main independent styles all possible changes in `fd-reset` will affect main component representation.

Anyway, If you think that I should remove it anyway - I will remove it, but I think this could case issues in the future.